templates/collection_view_screen/app/screens/name_screen.rb

```diff
@@ -0,0 +1,59 @@
+class <%= @name_camel_case %>Screen < UICollectionViewController
+  include ProMotion::ScreenModule
+
+  title "Your title here"
+  stylesheet <%= @name_camel_case %>ScreenStylesheet
+
+  <%= @name.upcase %>_CELL_ID = "<%= @name_camel_case %>Cell"
+
+  def self.new(args = {})
+    # Set layout
+    layout = UICollectionViewFlowLayout.alloc.init
+    s = self.alloc.initWithCollectionViewLayout(layout)
+    s.screen_init(args) if s.respond_to?(:screen_init)
+    s
+  end
+
+  def on_load
+    collectionView.tap do |cv|
+      cv.registerClass(ImagesCell, forCellWithReuseIdentifier: IMAGES_CELL_ID)
+      cv.delegate = self
+      cv.dataSource = self
+      cv.allowsSelection = true
+      cv.allowsMultipleSelection = false
+      rmq(cv).apply_style :collection_view
+    end
+  end
+
+  # Remove if you are only supporting portrait
+  def supportedInterfaceOrientations
+    UIInterfaceOrientationMaskAll
+  end
+
+  # Remove if you are only supporting portrait
+  def willAnimateRotationToInterfaceOrientation(orientation, duration: duration)
+    rmq(:reapply_style).reapply_styles
+  end
+
+  def numberOfSectionsInCollectionView(view)
+    1
+  end
+
+  def collectionView(view, numberOfItemsInSection: section)
+    200
```

Comment:
Derek, for the supportedInterfaceOrientations and willAnimateRotationToInterfaceOrientation, please use the ProMotion versions of these.